spec/requests/products_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Products', type: :request do
```

Comment:
RSpec.describe should have the class name you are testing. Do you have a products.rb class?

So you should have it like:

`RSpec.describe PasswordResetsController, type: :request `

By having it like this, you will be able to call described_class helper method which will then be the instance of PasswordResetsController